sys-libs/libcxx/libcxx-9999.ebuild

```diff
@@ -9,13 +9,13 @@ EGIT_REPO_URI="http://llvm.org/git/libcxx.git
 	https://github.com/llvm-mirror/libcxx.git"
 CMAKE_MIN_VERSION=3.4.3
 
-[ "${PV%9999}" != "${PV}" ] && SCM="git-r3" || SCM=""
+[[ ${PV} == 9999 ]] && SCM="git-r3" || SCM=""
```

Comment:
Well, I can write these 4 steps in 4 lines of C with O(|input|) complexity. I doubt you can say the same for pattern matching.

You might want to read: https://www.sourceware.org/ml/libc-alpha/2007-03/msg00023.html

or run this so pretty pattern matching:
```
a="aaaabbbbccccddddeeeeffffgggghhhhiiiijjjjkkkkllllmmmmnnnnooooppppqqqqrrrrssssttttuuuuvvvvwwwwxxxxyyyy"
[[ $a == a*b*d*e*f*g*h*i*j*k*l*m*n*o*p*q*z* ]] && echo yes
```

If we paid more attention to bash code at global scope, maybe metadata generation or emerge -uND world wouldnt be so slow :)

(I didnt check, but if bash processes endswith as a pattern then there is room for improvement there)